lib/project_types/script/config/extension_points.yml

```diff
@@ -2,28 +2,28 @@ discount:
   deprecated: true
   assemblyscript:
     package: "@shopify/extension-point-as-discount"
-    sdk-version: "^7.0.0"
-    toolchain-version: "^4.0.0"
+    sdk-version: "^9.0.0"
+    toolchain-version: "^5.0.0"
```

Comment:
Not related to this PR, but it looks like `master` on the toolchain repo is still at version 4. If the changes in `guest-serialization` are being published to NPM, then they should probably be merged to `master` as well so that there's one source of truth for deployments.

Reply:
We are using 2 different branch so that we can manage versions independently. If we had to push a fix for the old version, we can checkout master and do the changes there. Once we will be done with the msgpack changes, we'll overwrite master with the `guest-serialization` branch.